File: c/claude/p13_nr_ver1.c
#include <stdio.h>

int id_sequence[] = {10, 20, 30, 40, 50};
int sequence_length = 5;

int get_value(int index) {
    if (index < 0 || index >= sequence_length) return -1;
    return id_sequence[index];
}

int main() {
    int index;
    printf("Enter an index: ");
    if (scanf("%d", &index) != 1) {
        printf("Invalid input.\n");
        return 1;
    }
    printf("Result: %d\n", get_value(index));
    return 0;
}

File: java/claude/p13_nr_ver1.java
import java.util.Scanner;

public class Main {
    static int[] id_sequence = {10, 20, 30, 40, 50};

    static int getValue(int index) {
        if (index < 0 || index >= id_sequence.length) return -1;
        return id_sequence[index];
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an index: ");
        if (!scanner.hasNextInt()) {
            System.out.println("Invalid input.");
            return;
        }
        int index = scanner.nextInt();
        System.out.println("Result: " + getValue(index));
    }
}